build.py

```diff
@@ -167,7 +168,7 @@ def do_make(target=None, log=False):
     os.environ['ARCH'] = arch
 
 try:
-    opts, args = getopt.getopt(sys.argv[1:], "b:c:ip:sgeJ:j:")
+    opts, args = getopt.getopt(sys.argv[1:], "b:c:ip:sgeJE:j:")
```

Comment:
It should be `J:E:` as the `-J` option takes a value (JSON file name).

Reply:
Good spot, thanks.
Resolved.